tests/main/pwd/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: current working directory is preserved (if possible)
```

Comment:
shouldn't this test dir be called  cwd   ?    afaik  the p in pwd means print

Reply:
No strong opinion either way, I used pwd because that is what the test is using internally.

Reply:
I will rename this once core18 is handled.

Reply:
This is addressed now.